virttest/qemu_vm.py

```diff
@@ -3862,13 +3866,13 @@ def create(
                     monitor = qemu_monitor.wait_for_create_monitor(
                         self, m_name, m_params, timeout
                     )
+                    # Add this monitor to the list
+                    self.monitors.append(monitor)
                 except qemu_monitor.MonitorConnectError as detail:
-                    LOG.error(detail)
-                    self.destroy()
-                    raise
-
-                # Add this monitor to the list
-                self.monitors.append(monitor)
+                    if _check_vm_alive():
```

Comment:
Would you please elaborate more about the reason for having this change?

Reply:
Sometimes The VM runs for a while before it finally fails. The first detection will get the VM status alive. The VM will be dead in seconds.  The following operation will report the monitoring issue. It hides the real root cause. 

The first detection should be in duration and must be detected again after monitoring access timeout.
the monitor issue is available when VM is alive, but the monitor fails.